src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -448,9 +483,11 @@ void ClusterResourceScheduler::DeleteResource(const std::string &node_id_string,
       local_view->custom_resources.erase(itr);
     }
 
-    if (node_id == local_node_id_) {
+    auto ltr = local_resources_.custom_resources.find(resource_id);
```

Comment:
What is ltr? Can you use a different variable name here?

Reply:
ok,